### DISCUSSION TITLE
Stops in the process - No 'Access-Control-Allow-Origin' header is present on the requested resource.

### DISCUSSION THREAD

**Discussion by JWomm** [davetapley/helium-tax#38] [General]
Hi - I've been using this tool once before - but now, while processing the data-request it stops in the middle unfortunately - I see following console errors
![grafik](https://user-images.githubusercontent.com/5838478/141694550-a6e7b20a-02dc-47ac-8971-28e82020de23.png)

Maybe there are too many requests? Maybe it'd be helpful offering also the selection of the month to reduce requests

**Comment by davetapley** [comment]
Thanks for reporting @JWomm.

I _believe_ this is caused by ongoing outage (per https://status.helium.com/), so 🤞🏻 it'll resolve itself when that does.

I've also got https://github.com/davetapley/helium-tax/issues/26 in the pipeline to better detect/show when this happens.
